WeakAurasOptions/AceGUI-Widgets/AceGUIWidget-WeakAurasDisplayButton.lua

```diff
@@ -944,6 +1026,11 @@ local methods = {
     self.ungroup:SetScript("OnClick", self.callbacks.OnUngroupClick);
     self.upgroup:SetScript("OnClick", self.callbacks.OnUpGroupClick);
     self.downgroup:SetScript("OnClick", self.callbacks.OnDownGroupClick);
+
+    if WeakAurasWagoUpdate and not self.data.ignoreWagoUpdate then
+      self.callbacks.ShowWagoUpdate()
```

Comment:
This is the initialization function of a singular button. ShowWagoUpate() recursively updates all child buttons, thus any child button is initialized twice. Also the code above to initialze the menu is overwritten by ShowWagoUpdate() in some cases.

Also ShowWagoUpdate() does various things that aren't truely needed on each Show/Hide toggling. E.g. the lookup of the slug and the title/description setup.

I think you might be better to move some of the code from Show e.g. the lookup of of the slug to this initialize function and to not call ShowWagoUpdate here.